dks-host/dks-host-install.sh

```diff
@@ -44,6 +44,8 @@ echo "JAR_DOWNLOAD_URL: $URL"
 curl "$URL" -L -o /tmp/dks.jar
 sudo useradd dks -m
 
+sudo echo ${VERSION} > /opt/dks/version
```

Comment:
sudo and redirect won't work :)

Reply:
OK, so as you noted before. Does Packer actually require sudo for every step of its script?

Reply:
I haven't tested it without sudo and previous code had sudo. If it runs under ec2-user or whatever then I'd expect it to need sudo. In other words, I don't know :)

Reply:
OK, have pushed it to test :)